crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -63,15 +63,15 @@ impl VersionAndIdPdu {
         })
     }
 
-    pub fn encode(&self, dst: &mut WriteCursor) -> PduResult<()> {
+    pub fn encode(&self, dst: &mut WriteCursor<'_>) -> PduResult<()> {
         ensure_size!(ctx: self.name(), in: dst, size: Self::FIXED_PART_SIZE);
         dst.write_u16(self.version_major);
         dst.write_u16(self.version_minor);
         dst.write_u32(self.client_id);
         Ok(())
     }
 
-    pub fn decode(header: SharedHeader, payload: &mut ReadCursor) -> PduResult<Self> {
+    pub fn decode(header: SharedHeader, src: &mut ReadCursor<'_>) -> PduResult<Self> {
```

Comment:
cc @ibeckermayer 
I remembered [you were favorable to using `src` for consistency](https://github.com/Devolutions/IronRDP/pull/199#discussion_r1342265487) with the rest of the codebase so I figured it was fine if I search/replaced the remaining occurrences. This code was probably already in-flight when you commented back.